src/main/clojure/clara/benchmark/accum_repeatedly_not_selected.clj

```diff
@@ -0,0 +1,16 @@
+(ns clara.benchmark.accum-repeatedly-not-selected
+  "Added to test performance improvements in clara-rules issue 182"
+  (:require [clara.rules :refer :all]
+            [clara.rules.accumulators :as acc])
+  (:import [clara.benchmark.beans Discount Customer]))
+
+(def max-id (acc/min (fn [^Customer c] (.getId c)) :returns-fact true))
+
+(defrule vip-discount
+  [?customer <- max-id :from [Customer]]
+  =>
+  (apply insert! (repeat 1000 (Discount. 10))))
```

Comment:
Agreed, given the small numbers we probably should fall back to something more performant on the Clojure side.  Will change.